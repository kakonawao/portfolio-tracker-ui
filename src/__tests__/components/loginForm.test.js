import React from "react";
import { shallow } from "enzyme";

import LoginForm from "../../components/loginForm";


describe("<LoginForm>", () => {
    it("renders form", () => {
        const props = {
            username: jest.fn(),
            password: jest.fn(),
            login: jest.fn()
        };

        const wrapper = shallow(<LoginForm {...props}/>);
        expect(wrapper).toMatchSnapshot();
    });
});
