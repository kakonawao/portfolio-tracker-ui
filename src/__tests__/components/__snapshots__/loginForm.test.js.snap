// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginForm> renders form 1`] = `
<form>
  <div>
    <label>
      Username
      <input
        name="username"
        type="text"
      />
    </label>
    <label>
      Password
      <input
        name="password"
        type="password"
      />
    </label>
  </div>
  <div>
    <button
      onClick={[MockFunction]}
      type="submit"
    >
      Login
    </button>
  </div>
</form>
`;
